core-test/src/main/java/com/orange/ouds/core/test/OudsBadgeTest.kt

```diff
@@ -13,20 +13,39 @@
 package com.orange.ouds.core.test
 
 import com.orange.ouds.core.utilities.OudsPreviewableComponent
+import org.junit.experimental.runners.Enclosed
 import org.junit.runner.RunWith
 import org.junit.runners.Parameterized
 
-@RunWith(Parameterized::class)
-class OudsBadgeTest(parameter: Any) : OudsComponentSnapshotTest(
-    OudsPreviewableComponent.Badge,
-    parameter,
-    OudsComponentTestSuite.theme,
-    OudsPreviewableComponent.Badge.PreviewWidthDp
-) {
-
-    companion object {
-        @JvmStatic
-        @Parameterized.Parameters
-        internal fun data() = OudsPreviewableComponent.Badge.parameters
+@RunWith(Enclosed::class)
+class OudsBadgeTest {
+
+    @RunWith(Parameterized::class)
+    class Default(parameter: Any) : OudsComponentSnapshotTest(
+        OudsPreviewableComponent.Badge.Default,
+        parameter,
+        OudsComponentTestSuite.theme
```

Comment:
`OudsPreviewableComponent.Badge.PreviewWidthDp` should be used here and below in `WithIcon`, otherwise "ExtraSmall" is cut in the snapshots:

<img width="1080" height="960" alt="com orange ouds core test_OudsBadgeTest$Default_takeDarkThemeSnapshot 0" src="https://github.com/user-attachments/assets/c2f2aa5f-10b3-4e5b-8cd4-5ab511035009" />